apps/meteor/ee/server/lib/audit/endpoints.ts

```diff
@@ -0,0 +1,76 @@
+import type { IUser } from '@rocket.chat/core-typings';
+import { Rooms } from '@rocket.chat/models';
+import type { PaginatedRequest, PaginatedResult } from '@rocket.chat/rest-typings';
+import Ajv from 'ajv';
+
+import { API } from '../../../../app/api/server/api';
+import { getPaginationItems } from '../../../../app/api/server/helpers/getPaginationItems';
+import { findUsersOfRoom } from '../../../../server/lib/findUsersOfRoom';
+
+const ajv = new Ajv({
+	coerceTypes: true,
+});
+
+type AuditRoomMembersParams = PaginatedRequest<{
+	roomId: string;
+	filter: string;
+}>;
+
+const auditRoomMembersSchema = {
+	type: 'object',
+	properties: {
+		roomId: { type: 'string' },
+		filter: { type: 'string', nullable: true },
+		count: { type: 'number', nullable: true },
+		offset: { type: 'number', nullable: true },
+		sort: { type: 'string', nullable: true },
+	},
+	required: ['roomId'],
+	additionalProperties: false,
+};
```

Comment:
I'm quite confused about what is the best approach here, but I thought we could define this under the rest-typings package. I see that we do this in some EE endpoints, but for some of them we define the schema in the EE folder as you did, so I'm not sure
My personal preference would be to define it within the package since we already have ajv instantiated there and a bunch of EE endpoints iinw

@MarcosSpessatto any preference here?

Reply:
The idea is to not to have these on core typings/rest typings cause they're EE.

There was some Tasso's PR to moce the majority time ago.